packages/codebytes/src/types.ts

```diff
@@ -1,16 +1,17 @@
+import { BackgroundProps } from '@codecademy/gamut-styles';
 import { UserClickData } from '@codecademy/tracking';
 
 import { LanguageOption } from './consts';
 
 export interface CodebytesChangeHandler {
   (text: string, language: LanguageOption): void;
 }
-export interface CodeByteEditorProps {
+export interface CodeByteEditorProps
+  extends Omit<BackgroundProps, 'onCopy' | 'bg'> {
```

Comment:
Totally fine to keep!, but curious why we need `onCopy` here? Is it an event on the background component?